util/updateXML/update1.7.0.0.py

```diff
@@ -42,3 +42,24 @@ def update(jobs):
         )
         for unit_dynamic_model in unit_dynamic_models:
             unit_dynamic_model.set_name("Dynawo.Electrical.Controls.Machines.VoltageRegulators.Standard.SCRX1")
+
+    modelica_models = jobs.dyds.get_modelica_models(lambda modelica_model: modelica_model.get_id() == "Dynawo.Electrical.Controls.Machines.VoltageRegulators.Standard.IEEET1")
+    for modelica_model in modelica_models:
+        unit_dynamic_models1 = modelica_model.get_unit_dynamic_models(lambda unit_dynamic_model: unit_dynamic_model.get_id() == "voltageRegulator")
+        for unit_dynamic_model1 in unit_dynamic_models1:
+            unit_dynamic_model1.init_connects.change_var_name("UStator0Pu", "Us0Pu")
```

Comment:
It is a parameter , I think we  can use something similar to that : (like what we did with HVDC)

IEEET1 = job.dyds.get_bbms(lambda bbm: bbm.get_lib_name() == "IEEET1")
for voltageregulator in IEEET1:
        value =voltageregulator.parset.get_param_value("voltageregulator_UStator0Pu")
        voltageregulator.parset.remove_param_or_ref("voltageregulator_UStator0Pu")
        voltageregulator.parset.add_param("DOUBLE", "voltageregulator_Us0Pu", value)

to be tested

Reply:
UStator0Pu appears in initConnect statements, the structure used for HVDC parameters is aimed at renaming parameters in .par.

Reply:
Okay could you add also one of of model_template?

Reply:
Done

Reply:
The test nrt-update 1.6.0 to 1.7.0 is successful.